lib/modules/hosts/superstack.js

```diff
@@ -0,0 +1,42 @@
+import { ajax } from '../../environment';
+
+const IMAGE = 'IMAGE';
+const GALLERY = 'GALLERY';
+
+const superstack = {
+	moduleID: 'superstack',
+	name: 'superstack',
+	domains: ['superstack.io'],
+	logo: '//superstack.io/favicon.ico',
+	detect({ href }) {
+		return (/^https?:\/\/(?:www\.)?superstack\.io\/v\/(\w+)(?:[/#]|$)/).exec(href);
+	},
+	async handleLink(href, [, id]) {
+		const data = await ajax({
+			url: `https://superstack.io/api/v/${id}`,
+			type: 'json',
+		});
+
+		if (data.error) {
+			throw new Error(`Super Stack API error: ${data.error}`);
+		} else if (!data.photos || data.photos.length < 1) {
+			throw new Error('Super Stack API error: no photos returned');
+		}
+
+		// Check if this is a single image or an album
+		if (data.photos.length === 1) {
+			return {
+				type: IMAGE,
+				src: data.photos[0].source,
+			};
+		} else {
+			return {
+				type: GALLERY,
+				title: data.albumTitle || data.photos[0].title,
+				src: data.photos.map(photo => photo.source),
```

Comment:
Individual images can have a `title` and `caption`, and should have a `type` ([see imgur](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/9644a2a0e49d8005c0b98c2af4ea060d2c30d20b/lib/modules/hosts/imgur.js#L181-L185); the docs can probably be improved)